### PR TITLE
Revert "https://issues.redhat.com/browse/ACM-9807--removals"

### PR DESCRIPTION
**Pull request by swopebe**
Reverts stolostron/rhacm-docs#6159

### CODE REVIEW

**Review by gparvin — COMMENTED** [review]
Why are the removals being reverted?  Even if you want to keep them, could we at least get rid of the duplicates?

File: release_notes/known_issues_governance.adoc [hunk 1]
```diff
@@ -40,6 +40,12 @@ When you use an external identity provider to log in to {product-title-short}, y
 
 You must log out of the external identity provider before you attempt to log out of {product-title-short}. 
 
+[#gatekeeper-upgrade]
+== Gatekeeper operator installation fails
+//2.4:16673
+
+When you install the gatekeeper operator on {ocp} version 4.9, the installation fails. Before you upgrade {ocp-short} to version 4.9.0., you must upgrade the gatekeeper operator to version 0.2.0. See link:../governance/create_gatekeeper.adoc#upgrading-gatekeeper-gatekeeper-operator[Upgrading gatekeeper and the gatekeeper operator] for more information.
```
Comment (dockerymick) [inline release_notes/known_issues_governance.adoc:47]: Gatekeeper must be capitalized
```suggestion
When you install the Gatekeeper operator on {ocp} version 4.9, the installation fails. Before you upgrade {ocp-short} to version 4.9.0., you must upgrade the Gatekeeper operator to version 0.2.0. See link:../governance/create_gatekeeper.adoc#upgrading-gatekeeper-gatekeeper-operator[Upgrading Gatekeeper and the Gatekeeper operator] for more information.
```

**Review by dockerymick — APPROVED** [review]
One suggestion, capitalize Gatekeeper

**Comment by swopebe** [comment]
No, this was my attempt to reopen the PR, not re-add the content. All of this is being removed.
So @gus just confirm and I will redo the original PR.

**Pull request closed without merging**